rules/windows/process_creation/win_susp_conhost.yml

```diff
@@ -0,0 +1,25 @@
+title: Conhost Parent Proces Executions
+id: 7dc2dedd-7603-461a-bc13-15803d132355
+status: experimental
+description: Detects the conhost execution as parent process. Can be used to evaded defense mechanism.
+references:
+    - http://www.hexacorn.com/blog/2020/05/25/how-to-con-your-host/
+author: omkar72
+date: 2020/10/25
+tags:
+    - attack.defense_evasion
+    - attack.t1202
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        ParentImage: '*\conhost.exe'
```

Comment:
Parent `conhost.exe`? 
I cannot estimate how many FPs that would cause.

Reply:
I believe conhost is launched by lot of processes for Gui purposes. But conhost with child process should be rare. This would be use case where specifically conhost is use to launch application through cmd/ps. Let me know what you think